feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/ui/QuoteCard.kt

```diff
@@ -0,0 +1,210 @@
+  quote: MoveQuote,
+  onExpandClicked: () -> Unit,
+  isExpanded: Boolean,
+) {
+  Column(
+    modifier = Modifier
+      .padding(horizontal = 16.dp)
+      .clip(SquircleShape)
+      .clickable { onExpandClicked() }
+      .background(shape = SquircleShape, color = Color(0xFFF0F0F0))
+      .padding(16.dp),
+  ) {
+    Row(verticalAlignment = Alignment.CenterVertically) {
+      Image(
+        painter = painterResource(id = R.drawable.ic_pillow),
+        contentDescription = "",
+        modifier = Modifier.size(48.dp),
+      )
+      Spacer(modifier = Modifier.padding(start = 16.dp))
+      Column {
+        Text(
+          text = "Hemförsäkring Bostadsrätt",
+          style = MaterialTheme.typography.titleMedium,
+          fontSize = 18.sp,
+        )
+        Row(verticalAlignment = Alignment.CenterVertically) {
+          Text(
+            text = "Aktiveras $movingDate",
+            style = MaterialTheme.typography.titleMedium,
+            color = Color(0xFF727272),
+            fontSize = 18.sp,
+          )
+          Spacer(modifier = Modifier.padding(start = 5.dp))
+          Icon(
+            painter = painterResource(id = com.hedvig.android.core.designsystem.R.drawable.ic_info),
+            tint = Color(0xFF727272),
+            contentDescription = "",
+          )
+        }
+      }
+    }
+
+    Spacer(modifier = Modifier.padding(top = 16.dp))
+
+    Divider(
+      thickness = 1.dp,
+      color = Color(0xFFCFCFCF),
+    )
+
+    Spacer(modifier = Modifier.padding(top = 8.dp))
+
+    Row(
+      modifier = Modifier.fillMaxWidth(),
+      horizontalArrangement = Arrangement.Absolute.SpaceBetween,
+      verticalAlignment = Alignment.CenterVertically,
+    ) {
+      Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier.padding(vertical = 8.dp),
+      ) {
+        Text(
+          text = "Detaljer",
+          style = MaterialTheme.typography.titleMedium,
+          fontSize = 18.sp,
+        )
+        Spacer(modifier = Modifier.padding(4.dp))
+
+        if (isExpanded) {
+          Icon(
+            painter = painterResource(id = com.hedvig.android.core.designsystem.R.drawable.ic_drop_down_indicator_expanded),
+            contentDescription = "",
+            tint = Color(0xFFB4B4B4),
+          )
+        } else {
+          Icon(
+            painter = painterResource(id = com.hedvig.android.core.designsystem.R.drawable.ic_drop_down_indicator),
+            contentDescription = "",
+            tint = Color(0xFFB4B4B4),
+          )
+        }
+      }
+
+      Text(
+        text = stringResource(
+          id = hedvig.resources.R.string.CHANGE_ADDRESS_PRICE_PER_MONTH_LABEL,
+          quote.premium.toString(),
+        ),
+        style = MaterialTheme.typography.titleMedium,
+        fontSize = 18.sp,
+      )
+    }
+
+    AnimatedVisibility(
+      visible = isExpanded,
+      enter = expandVertically(),
+      exit = shrinkVertically(),
+    ) {
```

Comment:
You *may* wanna do something like this https://github.com/HedvigInsurance/android/blob/eafa3492a8c9c2ce1bfb1971ef95d72a4000226c/feature-odyssey/src/main/kotlin/com/hedvig/android/feature/odyssey/step/singleitem/SingleItemDestination.kt#L129-L130 here, to also have the thing fade in as it's coming in, so it comes into the view more naturally, instead of immeditelly look like it's being clipped above.
The `clip = false` is maybe something you shouldn't bring over here, as it will most likely look odd as they come from above, but mostly taking about the `fadein() + expandVertically()` and the same for exit.

Reply:
Super nice!